initials.c

```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
+
+int main()
+{
+	printf("**********                           \n");
+	printf("*Sean Kee****************************\n");
+        printf("**********                        * *\n");
+        printf("*        ******   *****       ****  *\n");
+        printf("*     ******        ****     ***    *\n");
+        printf("*    *****          ***     ***     *\n");
+        printf("*    ****          ****   ***       *\n");
+        printf("*    *****         ***  ****        *\n");
+        printf("*     ******      ********          *\n");
+        printf("*       *****     *******           *\n");
+        printf("*        ****    **** ****          *\n");
+        printf("*       *****    ***   ****         *\n");
+        printf("*     ******    ****    ****  *     *\n");
+	printf("*   *****      ****      *****      *\n");
+	printf("*                                   *\n");
+	printf("*************************************\n");
+}
+
```

Comment:
Overall Assessment: Your code runs, but you should follow guidelines more closely in terms of commenting and returning the 0 at the end!